lib/manageiq/environment.rb

```diff
@@ -37,7 +37,7 @@ def self.ensure_config_files
     def self.while_updating_bower
       # Run bower in a thread and continue to do the non-js stuff
       puts "Updating bower assets in parallel..."
```

Comment:
@himdel I already merged but maybe the comment should move into the update_ui method or perhaps in the update:ui rake task itself?